src/net.md

```diff
@@ -26,7 +26,26 @@
 <a name="ex-url-new-from-base"></a>
 ## Create new URLs from a base URL
 
-[Write me!](https://github.com/brson/rust-cookbook/issues/35)
+[![url-badge]][url]
+```rust
+extern crate url;
+
+use url::{ParseError, Url};
+
+const BASE_GITHUB_URL: &'static str = "https://github.com";
+
+fn main() {
+    match build_github_url("rust-lang/cargo") {
```

Comment:
Please use 4 spaces for indentation inside the `match` block since this is what the Rust compiler codebase uses.